app/views/recipes/show.html.erb

```diff
@@ -0,0 +1,59 @@
+<div>
+  <% if flash[:notice] %>
+    <p class="alert alert-success" role="alert"><%= notice %></p>
+  <% end %>
+  <div class="card mb-3">
+    <div class="card-body">
+      <h5 class="card-title d-flex justify-content-center"><%= @recipe.name.capitalize %> recipe</h5>
+      <div class="d-flex justify-content-between m-2">
+        <div>
+          <p class="card-text">Description: <%= @recipe.description %></p>
+          <p class="card-text"><small class="text-muted">Cooking time: <%= @recipe.cooking_time %></small></p>
+          <p class="card-text"><small class="text-muted">Preparation time: <%= @recipe.preparation_time %></small></p>
+        </div>
+        <div class="form-check form-switch">
+          <% if can? :manage, @recipe %>
+            <%= form_with(model: @recipe) do |form| %>
+              <%= form.check_box :public,onchange: 'this.form.submit();',class:"form-check-input"%>
+              <label class="form-check-label" for="flexSwitchCheckDefault">public</label>
+            <% end %>
+            <div class="d-flex gap-2" style="margin:20px 0 0 -40px;"><%= link_to "Edit", edit_recipe_path(@recipe), class: "btn btn-outline-primary btn-sm" %>  <%= button_to "Destroy", @recipe, method: :delete, class: "btn btn-outline-danger btn-sm small-btn"%></div>
+          <% end %>
+        </div>
+      </div>
+    </div>
+  </div>
+  <div class="d-flex justify-content-between">
+    <%= link_to "🔙 Back to recipes", recipes_path, class: "btn btn-outline-secondary btn-sm"%>
+    <% if can? :manage, @recipe%>
+      <button class="btn btn-success" id="addIngredient" data-toggle="modal" data-target="#exampleModal" onclick="(() => {
+        document.getElementById('toggleModel').classList.toggle('d-none')
+        })()">Add ingredient</button>
+    <% else %>
+      <button class="btn btn-success" disabled id="addIngredient" data-toggle="modal" data-target="#exampleModal" onclick="(() => {
+        document.getElementById('toggleModel').classList.toggle('d-none')
+        })()">Add ingredient</button>
+    <% end %>
+  </div>
+  
+  <div class="border rounded-3 mt-3 p-3 ">
+    <div class=" align-items-center">
+      <h3 class="p-2">ingredients</h3>
+    </div>
+    <div id="shopping-list">
+      <table class="table">
+        <thead>
+          <tr>
+            <th scope="col">Food</th>
+            <th scope="col">Quantity</th>
+            <th scope="col">Price</th>
+          </tr>
+        </thead>
+        <tbody>
+          <%= render "shopping_lists/shopping_list" %>
+        </tbody>
+      </table>

```

Comment:
- [ ] Kindly add Action buttons, modify / remove in ingredients  you can set them as `disabled` 

Yous  | Wireframe
------------- | -------------
<img width="1315" alt="Screenshot 2022-05-27 at 18 33 06" src="https://user-images.githubusercontent.com/77038610/170720566-610edf2c-9331-408e-a8a8-4eac61670f3c.png">  | <img width="516" alt="Screenshot 2022-05-27 at 18 31 53" src="https://user-images.githubusercontent.com/77038610/170720345-05018f1b-43ac-47b3-998f-b652dc1564a1.png">